.storybook/recipes/GlobalHeader/GlobalHeader.module.css

```diff
@@ -49,17 +49,31 @@
   color: var(--eds-theme-color-text-neutral-default-inverse);
 
   &:hover,
-  &:focus {
+  &:focus-visible {
     background: var(
       --eds-theme-color-button-primary-brand-background-hover
     ); /* TODO */
     border-color: transparent; /* TODO */
   }
 
-  &:focus {
+  &:focus-visible {
     @mixin focusInverted;
   }
 
+  @supports not selector(:focus-visible) {
+    &:hover,
+    &:focus {
+      background: var(
+        --eds-theme-color-button-primary-brand-background-hover
+      ); /* TODO */
+      border-color: transparent; /* TODO */
```

Comment:
What were these `todo`s for?

Reply:
lol I have no idea. I mostly left comments and stuff alone since there's another ticket for that, but I can also clean them up while I'm in here. 🤷

Reply:
TODO: figure out what the todos were

Reply:
lol ?? I have a vague memory Ian was gonna update the tokens these point to or something. agree we can delete it

Reply:
I think it was this https://github.com/chanzuckerberg/edu-design-system/pull/955#discussion_r845273010 ... not sure why they didn't remove the TODO when they updated it next